### ISSUE TITLE
Comment

### ISSUE THREAD

**Issue by huhwhatthe** [chrismaltby/gb-studio#1267]
I've apparently been banned on discord for no obvious reason... is this something you're personally promoting? Are you happy with these petty dictator-wannabes? I even tried merely asking 'why'... and was 'disappeared' again within minutes. Like,  seriously? Are they too bored with gbs now, and instead want to harass people? I don't even know who's doing it, so congrats on the unbelievable amount of rationales and transparency...!

**Comment by maxoakland** [comment]
I agree. The mods on the discord are toxic and have created a very unhealthy environment. People have told me they felt the same and some have even left the discord because of it

I reached out to Chris about this awhile ago but I got no response, unfortunately

**Comment by WAUthethird** [comment]
@chrismaltby Highly recommend closing this issue and locking comments. Had this same user spewing vitriol and hate on Reddit, and after banning them,  recieved a extremely awful message in the form of a ban appeal. (which you can see for yourself if you're curious, as you're a mod)

**Comment by huhwhatthe** [comment]
> 

Oh, you just want to continue to be a petty, pathetic little dictator without anyone daring to say anything, do you? (What kind of message did you expect after you literally deleted an entire conversation YOU DIDN'T PARTICIPATE ONCE IN! OVER 80 REPLIES... NOT ONCE, YOU! No, you're just a pathetic, insecure, probably due to a micropenis, who can only destroy discussions and harass people for merely ask questions!)

 Who do you aspire to be most like, do tell, is it Hitler or Mussolini? Do you, like, have posters of them and jerk off to them too? You're utterly shameful, and as you can see I'm not the only one who thinks so too!

**Comment by maxoakland** [comment]
> @chrismaltby Highly recommend closing this issue and locking comments. Had this same user spewing vitriol and hate on Reddit, and after banning them, recieved a extremely awful message in the form of a ban appeal. (which you can see for yourself if you're curious, as you're a mod)

This user might be overreacting, but their point is true and I’m glad someone finally brought it up. It’s time for us to have a real conversation about the abusive modding on the GBS Discord

**Comment by patrickmollohan** [comment]
Sorry @maxoakland, but in this case, I agree with @WAUthethird. If there's a conversation to be had about the Discord moderation,
> No, you're just pathetic, and insecure, probably due to a micropenis

and

> Who do you aspire to be most like, do tell, is it Hitler or Mussolini?

isn't how to have it.

**Comment by huhwhatthe** [comment]
@patrickmollohan oh, I suppose it's to be had by DELETING THE DAMN CONVERSATION?!?

**Comment by aretrogamer** [comment]
- use tor browser
- create a new discord account

Act as if you're somebody else and make a completely different nickname and you're back in discord.
They ban you on IP address and/or discord account. 

Problem solved.
